storage/smiles/pom.xml

```diff
@@ -158,6 +158,16 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>uk.ac.ebi.beam</groupId>
+            <artifactId>beam-core</artifactId>
+            <version>RELEASE</version>
```

Comment:
@johnmay, if not mistaken the SMILES is only used in the tests, so, am I correct that this could better add <scope>test</scope?>

Reply:
yes but this in the 'smiles' module, something odd here. Like I said i think it's misscommit, also we control versions from the parent pom.xml, and RELEASE can cause problems.

Reply:
Yes this is a miscommit - not sure how it got in. I'll revert before the next round of commits